meson.build

```diff
@@ -1,33 +1,39 @@
-# Define project name
-project ('com.github.timecraft.js-test', 
-    'vala', 'c',
-    version: '0.1.0'
-)
```

Comment:
> I personally find all of this on one line to be a bit harder to follow and not as nice
Like with this, you can see it goes:
Name
Language(s)
Version

Especially with the version number, it makes it a bit easier for me to go in, quickly find it, and edit it when there is a new version number.